src/openloco/graphics/gfx.cpp

```diff
@@ -16,7 +21,75 @@ namespace openloco::gfx
     // 0x0044733C
     void load_g1()
     {
-        call(0x0044733C);
+        auto g1Path = environment::get_path(environment::path_id::g1);
+
+        std::ifstream stream(g1Path.make_preferred().u8string().c_str(), std::ios::in | std::ios::binary);
```

Comment:
ifstream takes a `std::path`, so you should just be able to do:
`std::ifstream stream(g1Path, std::ios::in | std::ios::binary);`

Reply:
Yep, victim of copy-paste from `fopen` - will fix.

Reply:
Fixed.